modules/docs/llm/upgrade-guides/15.0-UPGRADE-GUIDE.md

```diff
@@ -186,6 +187,24 @@ The promoted `SegmentedControl` includes several new features:
 </SegmentedControl>
 ```
 
+### Information Highlight ⚡️
```

Comment:
Why do we have emoji? Do we need it at all?

Reply:
was just trying to add some pazzas, i can remove it :(